tests/distributions/testing/test_gof.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import torch.distributions as dist
+
+from pyro.distributions.testing.gof import multinomial_goodness_of_fit
+
+
+def test_multinomial_goodness_of_fit():
```

Comment:
i'm confused about this test? does `print_histogram` not get called in existing tests?

Reply:
That's right printing is [disabled by default](https://github.com/pyro-ppl/pyro/blob/dc1351a8d44581aca8e32a29939c81777e981c2e/pyro/distributions/testing/gof.py#L86) (I think because it was filling up logs), and this new test explicitly enables printing to make sure it works.